### PR TITLE
SWIFT-1125 Add release script

### PR DESCRIPTION
**Pull request by kmahar**
Adds a release script for the library. This is extremely similar to our other release scripts, so hopefully easy to review 🙂

I tested this out with the lines that actually tag a release commented out to ensure the docs logic worked correctly. You can see resulting docs now for two (not actually tagged) versions here: https://mongodb.github.io/mongodb-vapor/

I figure I will just delete the directories for those once before we actually release this.

### CODE REVIEW

File: etc/release.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+# go back to wherever we started
+git checkout -
+
+# tag release and push tag
+git tag "v${version}"
+git push --tags
+
+# go to GitHub to publish release notes
+open "https://github.com/mongodb/mongodb-vapor/releases/tag/v${version}"
```
Comment (patrickfreed) [inline etc/release.sh]: I know this exists already in the other scripts, but `open` is actually macOS specific, so this line fails on Linux (`xdg-open` is the equivalent). So we could try to branch here based on OS and open it properly that way, or just echo the docs URL, either works in my mind but the latter is simpler.

**Comment by patrickfreed** [comment]
I'm not sure if this PR is the one to fix this in, but from looking at the generated docs, it seems like it'd be worth documenting the extensions we provide to `Request` and `Application`, since right now they just say "undocumented". (e.g. [`Request`](https://mongodb.github.io/mongodb-vapor/current/Extensions/Application.html)) Ditto for the top level ["Vapor Extensions"](https://mongodb.github.io/mongodb-vapor/current/Vapor%20Extensions.html) section if possible.

File: etc/release.sh (hunk 1 above)
Comment (kmahar) [reply etc/release.sh]: ooo TIL. agree just echoing is simpler, switched to that.

**Comment by kmahar** [comment]
good idea re documenting those, I opened SWIFT-1215

**Review by patrickfreed — APPROVED** [review]

**Pull request merged**